bindings/nodejs/lib/types/block/output/feature.ts

```diff
@@ -107,6 +110,30 @@ class TagFeature extends Feature {
     }
 }
 
+/**
+ * Native token feature.
+ */
+class NativeTokenFeature extends Feature {
+    /**
+     * Identifier of the native token.
+     */
+    readonly id: HexEncodedString;
+    /**
+     * Amount of native tokens of the given Token ID.
+     */
+    readonly amount: u256;
```

Comment:
Can you not use a `NativeToken` instead of the fields then ?

Reply:
Yeah but I mean, Isn't there a decorator to flatten? Like serde(flatten)?

Reply:
i could make a custom de-ser i guess, but i think thats a but of a time waste :)